src/kemal/config.cr

```diff
@@ -1,5 +1,9 @@
 module Kemal
-  VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
+  {% if flag?(:windows) %}
+    VERSION = {{ `shards version`.chomp.stringify }}
+  {% else %}
+    VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
+  {% end %}
```

Comment:
```suggestion
  VERSION = {{ `shards version "#{__DIR__}"`.chomp.stringify }}
```

Reply:
This does not work on Windows

Reply:
Why not?

Reply:
There you go

```
Error in ..\..\..\Uchiha: Expected SCALAR but was SEQUENCE_START at line 1, column 1

  1. [4418:4D10][2024-09-04T11:27:02]i001: Burn v3.10.4.4718, Windows v10.0 (Build 22631: Service Pack 0), path: C:\Windows\Temp\{69826437-B142-429C-A2D4-00F0F0137311}\.cr\VC_redist.x64.exe
     ^

Showing last frame. Use --error-trace for full trace.

In src\kemal\config.cr:3:18

 3 | VERSION = {{ `shards version #{__DIR__}`.chomp.stringify }}
                  ^
Error: error executing command: shards version C:\Users\Uchiha Sasuke\playground\kemal\src\kemal, got exit status 1
```

Reply:
Double quotes around path are missing.

Reply:
Oh, you're right 👍 I blame Cursor for that 🤣